include/gazebo_yarp_plugins/ControlBoardDriver.h

```diff
@@ -351,6 +355,13 @@ class yarp::dev::GazeboYarpControlBoardDriver:
     std::vector<GazeboYarpControlBoardDriver::PID> m_velocityPIDs;
     std::vector<GazeboYarpControlBoardDriver::PID> m_impedancePosPDs;
 
+
+    // std::map<std::string, gazebo::common::PID> m_gazeboPositionPIDs; /* position PID controllers for each joint, needed to call gazebo's setForce() method */
+    std::vector<gazebo::common::PID> m_gazeboPositionPIDs;
+    std::vector<gazebo::common::PID> m_gazeboVelocityPIDs; /* velocity PID controllers for each joint, needed to call gazebo's setForce() method */
+    std::vector<gazebo::common::PID> m_gazeboImpedancePosPIDs; /* impedance position PID controllers for each joint, needed to call gazebo's setForce() method */
```

Comment:
The PID is necessary for positions and velocity (`m_gazeboPositionPIDs` and `m_gazeboVelocityPIDs`) loops, that are implemented in the JointController. I was referring just to the `m_gazeboImpedancePosPIDs`, that is not needed until we replace the implementation in `sendImpPositionToGazebo`